README.md

```diff
@@ -156,6 +163,58 @@ validators include:
 | MinLength(n) |   string        |   Enforces that a response is at least the given length       |
 | MaxLength(n) |   string        |   Enforces that a response is no longer than the given length |
 
+## Help Text
+
+All of the prompts have a `Help` field which can be defined to provide more information to your users:
```

Comment:
Minor nitpick, `Password` does not have `Help`, but not sure it is worth calling out.